### DISCUSSION TITLE
panos_ike_crypto_profile doesn't support Diffie-Hellman group 21

### DISCUSSION THREAD

**Discussion by bmeyer144** [PaloAltoNetworks/pan-os-ansible#452] [Q&A] [answered]
Latest PANOS supported Diffie-Hellman group, group21 is not an allowed entry in the Ansible module. 

"msg": "value of dh_group must be one or more of: group1, group2, group5, group14, group19, group20. Got no match for: group21"

**Comment by jamesholland-uk** [comment, marked as answer]
Hi @bmeyer144, we'll track this in https://github.com/PaloAltoNetworks/pan-os-ansible/issues/460
